util.go

```diff
@@ -418,12 +418,26 @@ func holidaysCheck(u *User) {
 }
 
 func printPrettyDuration(d time.Duration) string {
```

Comment:
This makes the relative timestamps too long, I prefer the older format

Reply:
Understood, I will revert it back.